packages/config/config/devices/0x0330/zv2833pac.json

```diff
@@ -0,0 +1,211 @@
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV2833PAC",
+	"description": "Push-Button Coupler",
+	"devices": [
+		{
+			"productType": "0x0400",
+			"productId": "0xd105"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Launch 2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Launch 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Launch 4",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"label": "External Switch Type",
+			"description": "External Switch Type",
```

Comment:
```suggestion
```
We avoid descriptions that duplicate or merely restate the labels.
Same for several others.

Reply:
Done